app/controllers/concerns/nav_tab.rb

```diff
@@ -34,13 +35,15 @@ def admin_tab?
     ((self.class.admin_actions || []) & [action_name.to_sym, :all]).any?
   end
 
+  def manage_mode?
+    admin_tab? && current_facility.present? && current_facility != Facility.cross_facility
+  end
+
   def navigation_links
     case
-    when customer_tab? && acting_user.present?
```

Comment:
I kind of hosed the squash and merge of #1704 when I tried to fix something, so this includes the fixes for that (removal of the `link_collection.manager` case and the switch to `!actting_as`

Reply:
Looks like squashing the commits when you merged messed up my local and I didn't notice so I accidentally pushed dirty commits. It looks like it's all fixed now, though.